### PR TITLE
Add some fixes to the campaign Full

### PR DESCRIPTION
**Pull request by YosraAk**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x 
| Description?  | Add some fixes to the campaign Full
| Type?         |  refacto 
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | /
| How to test?  | Run this command: npm run full-test -- --URL=ShopURL

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11122)

### CODE REVIEW

File: tests/E2E/test/campaigns/full/02_product/14_options_in_catalog_page.js [hunk 1]
```diff
@@ -32,6 +32,14 @@ scenario('Check the options in the catalog page', () => {
...
         .then(() => client.switchWindow(1));
     });
+    test('should click on the "Preview" link', () => {
+      return promise
+        .then(() => {
+          if (global.ps_mode_dev) {
+            client.waitForExistAndClick(AddProductPage.preview_link)
+          }
+        })
+    });
```
Comment (Quetzacoalt91) [inline tests/E2E/test/campaigns/full/02_product/14_options_in_catalog_page.js]: I see this code written two times. You should move this code in a common secnario file, and call it from here.

File: tests/E2E/test/campaigns/full/02_product/10_check_product_page_buttons.js [hunk 2]
```diff
@@ -34,7 +34,14 @@ scenario('Check product page buttons', () => {
...
     test('should switch to the Front Office', () => client.switchWindow(1));
-    test('should click on the "Preview" link', () => client.waitForExistAndClick(AddProductPage.preview_link));
+    test('should click on the "Preview" link', () => {
+      return promise
+        .then(() => {
+          if (global.ps_mode_dev) {
+            client.waitForExistAndClick(AddProductPage.preview_link)
+          }
+        })
+    });
```
Comment (Quetzacoalt91) [inline tests/E2E/test/campaigns/full/02_product/10_check_product_page_buttons.js]: I see this code written two times. You should move this code in a common secnario file, and call it from here.

**Comment by YosraAk** [comment]
Hi @Quetzacoalt91 ,
Done. Thanks for your review.

**Review by fatmaBouchekoua — APPROVED** [review]

**Comment by fatmaBouchekoua** [comment]
@mbadrani this PR is ready for merge

**Pull request merged**